src/bot/eval.py

```diff
@@ -1,25 +1,30 @@
+"""
+This module contains functions
+for evaluating code using the Coliru API.
+"""
+
 import datetime
 import discord
 
 from discord.ext import commands
-from util import coliru
+from util import coliru  # pylint: disable=no-name-in-module
```

Comment:
Pylint complains that there is no name `coliru` in the `util` module, although it works flawlessly, so I disabled it.